eng/pipelines/preview.yml

```diff
@@ -0,0 +1,74 @@
+trigger: none
+
+pr: none
+
+parameters:
+- name: UseTypeSpecNext
+  displayName: Use TypeSpec "next" version
+  type: boolean
+  default: true
+
+resources:
+  repositories:
+    - repository: sdk-repository
+      type: github
+      endpoint: Azure
+      name: Azure/azure-sdk-for-python
+      ref: refs/heads/users/pahallis/cookiecutter
+    - repository: azure-sdk-tools
+      type: github
+      endpoint: Azure
+      name: Azure/azure-sdk-tools
+      ref: refs/heads/main
+
+variables:
+  PythonVersion: "3.10"
+
+pool:
+  name: azsdk-pool-mms-ubuntu-2004-general
+  vmImage: MMSUbuntu20.04
+
+extends:
+  template: eng/pipelines/templates/stages/archetype-autorest-preview.yml@azure-sdk-tools
+  parameters:
+    RegenerationJobCount: 1
+    BuildPrereleaseVersion: true
+    UseTypeSpecNext: ${{ parameters.UseTypeSpecNext }}
+    EmitterPackageJsonPath: packages/typespec-python/package.json
```

Comment:
Should we use this package.json or the one in `azure-sdk-for-python/eng/emitter-package.json`

Reply:
this parameter name is a little ambiguous.  In this context, we're referring to the package.json of the emitter package in the autorest repo.